### PR TITLE
Remove deprecated min function

### PR DESCRIPTION
**Pull request by MartyEwings**
> **Note:** **Deprecated** from Puppet 6.0.0, this function has been replaced with a
    built-in [`min`](https://puppet.com/docs/puppet/latest/function.html#min) function.
    DOC

### CODE REVIEW

**Review by LukasAud — APPROVED** [review]
LGTM. Waiting for a second review.

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**
